File: Go/uri_1099.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n = 0
	fmt.Scanln(&n)
	m := make([][2]int, n)
	sum := make([]int, n)

	// - input matrix
	for i := 0; i < len(m); i++ {
		for j := 0; j < 2; j++ {
			fmt.Scan(&m[i][j])
		}
	}

	// - soma de impares
	for i := 0; i < len(m); i++ {
		maxx := int(math.Max(float64(m[i][0]), float64(m[i][1])))
		minn := int(math.Min(float64(m[i][0]), float64(m[i][1])))
		for j := minn; j < maxx; j++ {
			if j%2 != 0 && j != minn && j != maxx {
				sum[i] += j
			}
		}
	}

	// - exibir
	for i := range sum {
		fmt.Println(sum[i])
	}
}

File: Python/uri_1099.py
v = []
s = 0
n = int(input())
for i in range(0, n):
    s = 0
    x, y = map(int, input().split(" "))
    big = max(x, y)
    small = min(x, y)
    for j in range(small + 1, big):
        if j % 2 != 0:
            s += j
    v.append(s)
for i in range(0, n):
    print(v[i])
